pysilcam/silcamgui/guicalcs.py

```diff
@@ -280,13 +288,13 @@ def silcview(datadir):
             im = load_image(filename, size)
 
         if zoom:
-            label = font.render('ZOOM [F]: ON', 1, (255, 255, 0))
+            label = font.render('ZOOM [F]: ON', 1, font_colour)
             im = pygame.transform.scale2x(im)
             screen.blit(im,(-size[0]/2,-size[1]/2))
         else:
            im = pygame.transform.scale(im, size)
            screen.blit(im,(0,0))
-           label = font.render('ZOOM [F]: OFF', 1, (255, 255, 0))
+           label = font.render('ZOOM [F]: OFF', 1, font_colour)
```

Comment:
E111: indentation is not a multiple of four  <!-- Automatically posted by Lintly -->